### DISCUSSION TITLE
how can I change select options inside table repeater

### DISCUSSION THREAD

**Discussion by RahelSiddiqi** [awcodes/filament-table-repeater#75] [Q&A]
how can I change select options inside the table repeater where this repeater is inside another filament repeater? 

like : 
```php
FilamentFormRepeater::make()->schema([
    //Some Form Elements
    FilamentTableRepeater::make()->schema([
           //Some Form Elements
    ])
])
```

Now I want to change the select element options inside FilamentTableRepeater when a select option changed in FilamentFormRepeater.

**Comment by awcodes** [comment]
Look into using $get and $set in the filament docs.

**Reply by RahelSiddiqi** [reply]
Inside table repeater if I use 
`->options(fn($get) => DB::query()->where('x_id', $get('dependent_id')->get())`
It is not updating.
NOTE: I did make dependent_id ->live().
If I get it out of TableRepeater same code works fine, and updating options.

Can you please possible make it little clearer.

**Reply by awcodes** [reply]
I would think you would need to traverse the state in the right way, since you are nested you might need something like `$get('../dependent_id')`.

The plugin uses Filament's repeater under the hood. It's just a different view.
